### DISCUSSION TITLE
Aeotec Multisensor 7 Vibration?

### DISCUSSION THREAD

**Discussion by jdiegmueller** [zwave-js/zwave-js#4833] [Q&A] [answered]
Should the Aeotec Multisensor 7 be reporting vibration?

I see there's three parameters relating to vibration -- 6, 7, and 87-- but I can't seem to get the sensor to generate a report.

Thank you for any insight.

**Comment by AlCalzone** [comment]
I don't have mine handy to test currently, but I think it generates a tamper notification or something like that.
Aeotec support should be able to tell you for certain.

**Comment by asucrews** [comment, marked as answer]
if I am reading there Engineering Spec correctly, 9 Notification: Home Security (0x07) will set to Tampering, product moved (0x09) when vibrating. That is only thing about vibration I can find.

The Engineering Spec is at the bottom.
https://aeotec.freshdesk.com/support/solutions/articles/6000232605-multisensor-7-user-guide
